.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @europace/pku-kitt
```

Comment:
GitHub zeigt mir hier einen Fehler an

Unknown owner on line 1: make sure the team @europace/pku-kitt exists, is publicly visible, and has write access to the repository

* @europace/pku-kitt

Reply:
Mir auch, ich habs aber nur aus den anderen kopiert 🤷‍♂️ Leider hab ich in der Orga auch keine Adminrechte, mglw. muss das Team hier noch als Contributor gesetzt werden oder sowas?

Reply:
ok dass wäre dann ein Folgetask => dann kann das hier ja erstmal so raus